inst/stan/epinowcast.stan

```diff
@@ -128,6 +135,11 @@ transformed parameters{
     imp_obs[k] = exp(imp_obs[k]);
   }
   }
+  // estimate share of cases with eventually known reference date, modeled as
+  // a first order random walk for each group on the logit scale
+  for (k in 1:g) {
+    alpha[k] = inv_logit(alpha_start[k] + cumulative_sum(alpha_epsilon[k]));
```

Comment:
I am using my vectorized random walk version here. May be abandoned later if we want to use more complex models for `alpha` to.

Reply:
yup this is the better implementation. For now, could also update the expectation model one as well.